statsmodels/tools/testing.py

```diff
@@ -16,8 +16,10 @@ def strip_rc(version):
 
 
 def is_pandas_min_version(min_version):
-    '''check whether pandas is at least min_version '''
-    return StrictVersion((pandas.__version__[:6])) >= min_version
+    '''check whether pandas is at least min_version
+    '''
+    from pandas import __version__ as pversion
+    return StrictVersion(strip_rc(pversion)) >= min_version
```

Comment:
> my point about the attitude was your way of checking is simply wrong.

Your very first example in the comment above is one that I gave to you as being *wrong*. I'll stop trying to clarify in my own words (apparently I'm not very good at it), but just copy from the PEP that talks about this (https://www.python.org/dev/peps/pep-0386/):

    The LooseVersion class is quite lax.
    ...
    This class makes any version string valid, and provides an algorithm to 
    sort them numerically then lexically.
    ...
    The problem with this is that while it allows expressing any nesting level it
    doesn't allow giving special meaning to versions (pre and post-releases
    as well as development versions), as expressed in requisites 2, 3 and 4.